src/checkers/periodicAwsChecker.go

```diff
@@ -74,9 +76,15 @@ func (p *PeriodicAwsChecker) StartChecking() {
 			json.Unmarshal([]byte(secretString), &secretMap)
 
 			for key, value := range secretMap {
-				if strings.Contains(key, ".pem") {
+				if strings.Contains(key, p.awsKeyString) {
+					stringValue := value.(string)
+
+					if strings.HasPrefix(stringValue, "-----BEGIN CERTIFICATE-----") {
```

Comment:
i'm fine with this conversion. i don't think we need to be too concerned about perf here, but let's add a comment about this line about what we're doing?

also is the number of prefixed dashes consistent? i really don't know.